ruby/lib/helix_runtime/build_task.rb

```diff
@@ -21,7 +21,7 @@ def initialize(name = nil, gem_spec = nil)
 
     def init(name = nil, gem_spec = nil)
       @name = name
-      @build_path = "target/release"
+      @build_path = ENV["DEBUG_RUST"] ? "target/debug" : "target/release"
```

Comment:
Should we have a separate `build_root` property, since we overwrite `build_path` in our examples?